geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/CommonParams.scala

```diff
@@ -61,12 +61,7 @@ trait KerberosParams {
   var keytab: String = _
 }
 
-trait RequiredCredentialsParams extends PasswordParams {
-  @Parameter(names = Array("-u", "--user"), description = "Connection user name", required = true)
-  var user: String = _
-}
-
-trait OptionalCredentialsParams extends PasswordParams {
+trait CredentialsParams extends PasswordParams {
```

Comment:
Ok, why did the required username parameter go away?  Don't we have at least one DS that requires a username somewhere:)?

Reply:
nope :)